server/src/main/java/fr/uga/l3miage/tp4/models/CandidateEvaluationGridEntity.java

```diff
@@ -0,0 +1,35 @@
+package fr.uga.l3miage.tp4.models;
+
+import lombok.Getter;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+@Getter
+public class CandidateEvaluationGridEntity {
+    @Id
+    private Long sheetNumber;
+
+    @Getter
+    private double grade;
+
+    private LocalDateTime submissionDate;
+
+    @ManyToMany(mappedBy = "EvaluationGrids")
+    private Set<EvaluationCriteriaEntity> criterias;
```

Comment:
pas bidi donc pas de `mappedBy` 
```suggestion
    @ManyToMany
    private Set<EvaluationCriteriaEntity> criterias;
```